context.go

```diff
@@ -2,8 +2,10 @@ package log
 
 import "context"
 
+type logContextType string
+
 // ContextKeyLogFields is the key for the logging fields context value.
-const ContextKeyLogFields = "nrfta/go-log/Fields"
+const ContextKeyLogFields logContextType = "nrfta/go-log/Fields"
```

Comment:
I added the custom type here to better conform to the context package practices. It prefers a custom type so the risk of key collision is removed.